config.example.yaml

```diff
@@ -32,6 +32,13 @@ templates:
             - mediawiki/v1/content
           # - mediawiki/v1/mobile
           # - mediawiki/v1/revision-scoring
+          modules:
+            - name: dummy-mod
+              version: 0.0.1
+              type: file
+              path: mods/dummy
```

Comment:
> Should we make the mods/ prefix non-optional / implicit to encourage all modules to live in one place?

@gwicke  I was thinking the same thing. Since we allow absolute paths too, if there is a reason for not putting a module in mods/, one might recur to that option.